pilot/pkg/networking/core/listener_inbound.go

```diff
@@ -688,6 +702,31 @@ func reportInboundConflict(lb *ListenerBuilder, old inboundChainConfig, cc inbou
 	}
 }
 
+func buildInboundHBONEPassthroughChain(lb *ListenerBuilder) []*listener.FilterChain {
+	mtls := authn.MTLSSettings{
+		Port: 0,
+		Mode: model.MTLSDisable,
+	}
+	cc := inboundChainConfig{
+		port: model.ServiceInstancePort{
+			ServicePort: &model.Port{
+				Name: model.VirtualInboundListenerName,
+				// Port as 0 doesn't completely make sense here, since we get weird tracing decorators like `:0/*`,
```

Comment:
good descriptive comment; nice!

Reply:
:slightly_smiling_face:

Reply:
I will say I copy+pasted this haha. But I did write the original comment so I'll take it